pipelines/fbc-release/README.md

```diff
@@ -17,6 +17,11 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                    | Yes       | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | No        | -                                                               |
 
+### Changes in 1.10.0
+- the task `sign-index-image` now requires the `manifestListDigests` parameter set with the `indexImageDigests`
+  result from the task `add-fbc-contribution`.
```

Comment:
The task in the pipeline is called `add-fbc-contribution-to-index-image` (I don't know why it isn't the same as the defined task name).. It may make more sense to use that here but up to you

Reply:
True. makes more sense. updated